### PR TITLE
1601 by Inlead: Implement ding_item_cache to DDB CMS.

### PR DESCRIPTION
**Pull request by inlead**
http://platform.dandigbib.org/issues/1601

### CODE REVIEW

**Comment by guddo** [comment]
This PR is made for the 7.x-2.5.1 release.

File: ding2.info [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name = Ding 2
 description = Install a Ding based library site.
+version = 7.x-2.5.1
```
Comment (cableman) [inline ding2.info]: Version should no be part of the PR

**Comment by JrgenGNielsen** [comment]
Could you rebase the commits to fix the merge conflicts?

**Comment by guddo** [comment]
Should these tests be removed? /cc @kasperg @cableman @JrgenGNielsen

**Comment by DDBdrift** [comment]
https://platform.dandigbib.org/issues/1601 | Closed

**Pull request closed without merging**